README.md

```diff
@@ -116,6 +116,17 @@ export PG_URL="host=localhost port=5432 user=portal password=secret sslmode=disa
 
 More information and examples on [Postgres docs](https://www.postgresql.org/docs/current/static/libpq-connect.html#LIBPQ-CONNSTRING)
 
+## Generate ECDSA Key Pair
```

Comment:
This is fine for local development but we will need to deploy this with (secrets? in) k8s. cc @illegalprime @ehsanmasdar

Reply:
@bunsenmcdubbs What did you have in mind? We will need to read the key, but I believe it can be stored in memory since it's just bytes. We can also lower the length of the key (thankfully ECDSA is relatively forgiving with key length) if environmental variables have a character cap.